test/unit/schema/mssql.js

```diff
@@ -154,8 +158,10 @@ describe("MSSQL SchemaBuilder", function() {
       this.dropTimestamps();
     }).toSQL();
 
-    equal(1, tableSql.length);
-    expect(tableSql[0].sql).to.equal('ALTER TABLE [users] DROP COLUMN [created_at], [updated_at]');
+    equal(3, tableSql.length);
```

Comment:
Integration test showing the original problem and how this change fixing it is still missing. I just added docker setup for running mssql tests locally in docker, so it might help testing a bit in the future.